README.md

```diff
@@ -193,7 +200,9 @@ docker run -d -v data:/root/ -e BSK_SUBDOMAIN_CONFIG=/root/config.json -p 3000:3
 Root stores the sqlite database that the subdomain uses to queue registrations, and watch zonefiles for broadcasting. To test connectivity for this setup run the following curl command:
 
 ```bash
-$ curl http://localhost:3000/index | jq
+curl http://localhost:3000/index | jq
```

Comment:
For this one, we should leave it as is since it shows the command *and* the output.